tests/contrib/trace/test_plugin.py

```diff
@@ -33,6 +33,7 @@ def test__trace_plugin__valid_schema(mock_send_report, handler_with_trace, mock_
     handler({}, mock_context)
 
     assert_valid_schema(iopipe.report.report, optional_fields=[
+        'disk',
```

Comment:
You don't need this as an optional field

Reply:
The three tests were failing without it. So it is needed until #190 is merged.